client/homebrew/utils/versionHistory.js

```diff
@@ -11,28 +13,33 @@ const DEFAULT_HISTORY_SAVE_DELAYS = {
 	'5' : 2 * 24 * 60
 };
 
-const DEFAULT_GARBAGE_COLLECT_DELAY = 28 * 24 * 60;
+const HB_DB = 'HOMEBREWERY-DB';
+const HB_STORE = 'HISTORY';
 
-const HISTORY_SAVE_DELAYS = global.config?.historyData?.HISTORY_SAVE_DELAYS ?? DEFAULT_HISTORY_SAVE_DELAYS;
-const GARBAGE_COLLECT_DELAY = global.config?.historyData?.GARBAGE_COLLECT_DELAY ?? DEFAULT_GARBAGE_COLLECT_DELAY;
```

Comment:
I think the way you had this before was simpler; just set the HISTORY_SAVE_DELAYS and GARBAGE_COLLECT_DELAY once at this point, instead of partially defining it over multiple locations.

Reply:
Reverted to the local consts, rather than trying to use config values.

Reply:
Config values are fine if you really want them, but my point was you can assign these values here instead of setting it to the default here and then loading the config later. Just do the default and config handling all in one place.